### DISCUSSION TITLE
Application opening and closing animations do not work

### DISCUSSION THREAD

**Discussion by MMMMD7D** [LawnchairLauncher/lawnchair#4360] [General]
Pls help

**Comment by shobhit747** [comment]
> Pls help

What's your device android version? And which lawnchair version are you using?

**Reply by MMMMD7D** [reply]
Android 13 Lawnchair version 14 beta 2 
Animations do not work in old version lawnchair

**Comment by LynxCatTheThird** [comment]
Maybe you can try [QuickSwitch](https://github.com/skittles9823/QuickSwitch)?
